Api/Api.cs

```diff
@@ -857,7 +857,7 @@ public bool DownloadData(string filePath, string organizationId)
             // Make sure the link was successfully retrieved
             if (!dataLink.Success)
             {
-                Log.Error($"Api.DownloadData(): Failed to get link for {filePath}. " +
+                Log.Trace($"Api.DownloadData(): Failed to get link for {filePath}. " +
```

Comment:
Cool! We could potentially check if the expected data type is sparse or not, and make it an error if not